versions/3.1.1.md

```diff
@@ -199,25 +199,35 @@ Note that no aspect of implicit connection resolution changes how [URIs are reso
 
 ### Data Types
 
-Data types in the OAS are based on the types supported by the [JSON Schema Specification Draft 2020-12](https://tools.ietf.org/html/draft-bhutton-json-schema-00#section-4.2.1).
-Note that `integer` as a type is also supported and is defined as a JSON number without a fraction or exponent part.
-Models are defined using the [Schema Object](#schema-object), which is a superset of JSON Schema Specification Draft 2020-12.
+Data types in the OAS are based on the types defined by the [JSON Schema Validation Specification Draft 2020-12](https://datatracker.ietf.org/doc/html/draft-bhutton-json-schema-validation-00#section-6.1.1):
```

Comment:
Here we are using a markdown link -- should this be a ReSpec link instead?

Reply:
This is fine.

Direct section links don't seem possible with ReSpec, so we have special handling for RFC links with section fragments, converting them into a ReSpec document reference _plus_ a direct link to the RFC section.

Links to JSON Schema specs without a section fragment are turned into ReSpec document references, and one of them is sufficient for the spec to show up in Normative References.

Direct section links (with a fragment) are left untouched.